.github/workflows/ci-test.yml

```diff
@@ -253,6 +253,41 @@ jobs:
           payload-file: ${{ env.MESSAGE_PAYLOAD_FILE }}
           webhook: ${{ secrets.WEBHOOK_FLAKY_TESTS }}
 
+  docker-compose-test:
```

Comment:
this job needs to be referenced in the last job in this file to gate changes landing, follow the pattern there and you'll see this job visualized as input to the last job.   It'd be good to see it succeed/fail once as well.